### ISSUE TITLE
[Bug]: AssertionError: Inverse geocoding matrix chunks are not equal to interferogram chunks

### ISSUE THREAD

**Issue by benwright151** [mobigroup/gmtsar#27]
**Describe the bug**
Assertion error midway through geocode_parallel command. I am processing multiple sentinel scenes (20+) and merging the 3 subswaths before geocoding/ unwrapping. Seems to fail at the "build ll2ra transform" step. The problem seems to occur when particular sentinel-1 scenes are included in the stack - however there is no way to determine which scenes are causing the problem or why.

**To Reproduce**
Run merging and geocoding processing on the following datasets, data will run perfectly (full script attached)
[scriptExample.txt](https://github.com/mobigroup/gmtsar/files/11048363/scriptExample.txt)
:
- S1A_IW_SLC__1SDV_20160103T082126_20160103T082153_009327_00D7B4_6190.SAFE
- S1A_IW_SLC__1SDV_20160115T082126_20160115T082153_009502_00DCB7_2058.SAFE
- S1A_IW_SLC__1SDV_20160127T082126_20160127T082153_009677_00E1D8_4515.SAFE

Add the following two files to the stack for process to fail:
- S1A_IW_SLC__1SDV_20160208T082125_20160208T082152_009852_00E6DB_C86F.SAFE
- S1A_IW_SLC__1SDV_20160220T082120_20160220T082146_010027_00EC0A_8A04.SAFE

**System and software version:**
 - OS: Linux Mint (Cinnamon 21.1) using pygmtsar docker
 - GMTSAR version:

**Processing Log**
Full log attached below:
[log2.txt](https://github.com/mobigroup/gmtsar/files/11041067/log2.txt)

**Comment by mobigroup** [comment]
Please specify what Docker image and version do you use?

**Comment by benwright151** [comment]
REPOSITORY           TAG       IMAGE ID       CREATED        SIZE
mobigroup/pygmtsar   latest    6cd0e53e4011   3 months ago   4.4GB

**Comment by mobigroup** [comment]
Ok, I can reproduce the error. Will be fixed sometimes soon.

**Comment by mobigroup** [comment]
You need to remove "wavelength=400" argument for interferogram processing. Replace
```
sbas.intf_parallel(pairs, wavelength=400, func=decimator, n_jobs=6)
```
to
```
sbas.intf_parallel(pairs, func=decimator, n_jobs=6)
```

**Comment by mobigroup** [comment]
See for details https://github.com/gmtsar/gmtsar/issues/706

**Comment by mobigroup** [comment]
The issue is completely fixed in PyPI PyGMTSAR library 2023.4.1
